src/Icons/doc/index.rst

```diff
@@ -89,31 +87,30 @@ define the HTML attributes added to the ``<svg>`` element:
 Icon Sets
 ~~~~~~~~~
 
-Icon Sets many icon sets available, each with their own unique style and set of icons
-provide a wide range of icons for different purposes, while maintaining a consistent
-look and feel across your application.
-
-With more than 200,000 icons available, you can find the perfect icon for your project.
-
-Here are some of the most popular icon sets available:
-
-===================== ======= ========= ======= =======================
-Icon Set               Icons  License   Prefix       Example
-===================== ======= ========= ======= =======================
-`Octicons`_              700   MIT       oct        oct:checked
-`Bootstrap Icons`_       2000  MIT       bi         bi:check
-`Boxicons`_              1700  MIT       bx         bx:check
-`Flowbite`_              1000  MIT       flowbite   flowbite:check
-`FontAwesome Icons`_     7000  CC BY 4.0 fa        fa:check
-`Heroicons`_             300   MIT       hero       hero:check
-`Iconoir`_               1500  MIT       iconoir    iconoir:check
-`Ionicons`_              1300  MIT       ion        ion:check
-`Lucide Icons`_          1500  MIT       lucide     lucide:check
-`Material Design Icons`_ 5000  Apache 2  mdi        mdi:check
-`Octicons`_              200   MIT       oct        :oct:check
-`Phosphor Icons`_        7000  MIT       phosphor   phosphor:check
-`Tabler Icons`_          5200  MIT       tabler     tabler:check
-===================== ======= ========= ======= =======================
+There are many icon sets available, each with their own unique style and set of
+icons, providing a wide range of icons for different purposes, while maintaining
+a consistent look and feel across your application.
+
+With more than 200,000 icons available, you can find the perfect icon for your
+project. Here are some of the most popular icon sets available:
+
+========================= ======= ========== ========== ================
+Icon Set                    Icons License    Prefix     Example
+========================= ======= ========== ========== ================
+`Octicons`_                   700 MIT        oct        oct:checked
+`Bootstrap Icons`_           2000 MIT        bi         bi:check
+`Boxicons`_                  1700 MIT        bx         bx:check
+`Flowbite`_                  1000 MIT        flowbite   flowbite:check
+`FontAwesome Icons`_         7000 CC BY 4.0  fa         fa:check
+`Heroicons`_                  300 MIT        hero       hero:check
+`Iconoir`_                   1500 MIT        iconoir    iconoir:check
+`Ionicons`_                  1300 MIT        ion        ion:check
+`Lucide Icons`_              1500 MIT        lucide     lucide:check
+`Material Design Icons`_     5000 Apache 2   mdi        mdi:check
+`Octicons`_                   200 MIT        oct        :oct:check
```

Comment:
`:oct:check` -> `oct:check`

Reply:
Good (and checked/fixed the other prefixes)